src/components/RebuttalBtns.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+interface RebuttalBtnsProps {
+  setRebuttal: (value: string) => void;
+}
+
+export const RebuttalBtns = ({ setRebuttal }: RebuttalBtnsProps) => {
```

Comment:
Can you rename this component and file to `RebuttalToggle`?

Reply:
Done!